core/vm/actor/builtin/v0/init/init_actor.cpp

```diff
@@ -23,8 +23,9 @@ namespace fc::vm::actor::builtin::v0::init {
 
   ACTOR_METHOD_IMPL(Construct) {
     OUTCOME_TRY(runtime.validateImmediateCallerIs(kSystemActorAddress));
-    InitActorState state{};
-    state.network_name = params.network_name;
+    InitActorState state{.address_map = {runtime.getIpfsDatastore()},
```

Comment:
Have you checked other amt containers? (map, balanceTable etc.)

Reply:
Yes, they need the same initialisation, but all of these have already been implemented in different ways.

Reply:
Ok